charts/nfs-subdir-external-provisioner/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,14 @@
+{{ if gt .Values.replicaCount 1.0}}
```

Comment:
1. there is no reason to use float 1.0, 1 is enough and add space between 1 and `}}`
2. there need be a value to enable/disable PDB
3. nit: it is recommended to use `{{-` in such `if`s and `end`s